### PR TITLE
Small fix to cluster definition and pushing clinical summary to synapse

### PR DESCRIPTION
**Pull request by CamiloPosso**
The changes here are the following:

1) Changes to NMF_helper and NMF_summary: Fixed the way the clusters were being defined. Now we are using all the runs when determining cluster membership, rather than the run with the lowest residual value. The heatmaps annotations now look a little cleaner. Also added the "aliveStatus" indicator to the heatmaps. 

2) Added a short script to upload the latest clinical summary to synapse, in the dataProcessing folder. The clinical summary is on synapse under the "Genomics transcriptomics clinical" subfolder.

3) Renamed an old script to match new underscore style.

### CODE REVIEW

File: dataProcessing/uploading_clinical_summary.Rmd [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+
+
+We just received the latest sample summary table, containing a lot of annotation
+data, including allele ratios. 
+
+
+
+```{r loading data}
+path <- "../proteomics/data/BeatAML_complete_sample_summary/PNNL_clinical_summary_12_08_2021.xlsx"
```
Comment (sgosline) [inline dataProcessing/uploading_clinical_summary.Rmd:23]: where is this data? it should not be on the github repo...

Comment (CamiloPosso) [reply dataProcessing/uploading_clinical_summary.Rmd:23]: It's not on github, it was file in my local folders. This "Data" folder folder is excluded from this repo, I am just uploading this to synapse with this script. See here for the file:
https://www.synapse.org/#!Synapse:syn26532699

Comment (sgosline) [reply dataProcessing/uploading_clinical_summary.Rmd:23]: Ok, then please replace with a syn get. all code should be runnable for any of us with access.

Comment (CamiloPosso) [reply dataProcessing/uploading_clinical_summary.Rmd:23]: For clarity, uploading_clinical_summary.Rmd is there for document the upload of a new table summarizing clinical data for the patients to synapse. It's not meant to be run multiple times by someone with synapse access, as it needs the local file to push to synapse. I'll try to add a small comment to these kinds of scripts mentioning this from now.

**Pull request merged**